organization/_posts/2017-12-14-startup.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "HSF Weekly Meeting #120, December 14, 2017"
+layout: default
+---
+
+### *Present/Contributors*: Graeme Stewart, Benedikt Hegner, David Lange, Michel Jouvin, Liz Sexton-Kennedy, John Harvey
+
+### News, general matters
+
+-   ### HSF/WLCG Workshop
+
+    -   Meeting of Benedikt, Graeme, Ian Collier and Michel last
+        Tuesday: agreed to send a new email on Monday at the latest.
+    -   Working with local organizers to get logistical information
+        added to the website before Monday.
+    -   Decided to postpone opening the registration until mid-January to
+        get a chance to obtain some sponsorship and lower a
+        bit the registration fees (currently 300€ for early bird until
+        mid Feb; 340€ after early bird, but before the workshop; 380€
+        at the workshop).
+    -   Sorting out room availability, but probably no showstopper,
+        either on Monday morning (will start at 10 am but
+        registration will open before that) or on Thursday (the plan is
+        that the 150 seat room will be extended by Vidyo in a smaller
+        room if needed).
+    -   Liz: Hotel information is very important for US Lab people.
+        Fallback would be for us to gather information ourselves and
+        pass it on in the announcement next week.
+    -   First round at identifying conveners was done.
```

Comment:
not true. we listed names, but didn't contact any of them. Doesn't really qualify for *identifying*

Reply:
Fixed